### PR TITLE
[benchmark] rm minor warning

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

**Comment by dan-zheng** [comment]
Please submit non-`tensorflow`-branch specific changes to `master` branch!
I'll change the base branch to `master` branch now.

File: benchmark/single-source/DataBenchmarks.swift [hunk 1]
```diff
@@ -416,7 +416,7 @@ func withUnsafeMutableBytes(_ N: Int, data: Data) {
 @inline(never)
 func copyBytes(_ N: Int, data: Data) {
   let amount = data.count
-  var buffer = UnsafeMutablePointer<UInt8>.allocate(capacity: amount)
+  let buffer = UnsafeMutablePointer<UInt8>.allocate(capacity: amount)
```
Comment (dan-zheng) [inline benchmark/single-source/DataBenchmarks.swift:419]: @palimondo: could you please review this small change? I'm not sure whether usage of `var` was intentional.

**Comment by CodaFi** [comment]
LGTM

I doubt this will have an impact on the benchmarks, but it doesn't hurt to check.

**Comment by CodaFi** [comment]
@swift-ci please smoke test

**Comment by CodaFi** [comment]
@swift-ci please smoke benchmark

**Comment by CodaFi** [comment]
@swift-ci please smoke test

**Comment by CodaFi** [comment]
@swift-ci please smoke benchmark

**Comment by CodaFi** [comment]
@swift-ci please smoke benchmark

**Comment by CodaFi** [comment]
@swift-ci smoke test Linux

**Comment by CodaFi** [comment]
@swift-ci smoke benchmark

File: benchmark/single-source/DataBenchmarks.swift (hunk 1 above)
Comment (palimondo) [reply benchmark/single-source/DataBenchmarks.swift:419]: I don’t recall specific reason… if it complies, `let` Is fine :-)

**Comment by CodaFi** [comment]
Yeah, that’s noise.

**Comment by CodaFi** [comment]
@swift-ci smoke test macOS

**Comment by CodaFi** [comment]
⛵

**Pull request merged**